src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1814,11 +1824,24 @@ private static IOperation MakeInvalidOperation(ITypeSymbol type, IOperation chil
                                         constantValue: default, isImplicit: true);
         }
 
+        private static IOperation MakeInvalidOperation(SyntaxNode syntax, ITypeSymbol type, IOperation child1, IOperation child2)
+        {
+            return new InvalidOperation(ImmutableArray.Create<IOperation>(child1, child2),
+                                        semanticModel: null, syntax, type,
+                                        constantValue: default, isImplicit: true);
```

Comment:
Why are invalid operations implicit?

Reply:
> Why are invalid operations implicit? 

Usually, implicit operations are those that are synthesized. Like this one, for example. It is not a clone of a node present in the original tree. 

---
In reply to: [187123049](https://github.com/dotnet/roslyn/pull/26734#discussion_r187123049) [](ancestors = 187123049)